app/views/attendees/_attendee.html.erb

```diff
@@ -0,0 +1,5 @@
+<div>
+ <% if User.find_by(email: "#{attendee.email}") %>
+   <%= User.find_by(email: "#{attendee.email}").name %>
+ <% end %>
+</div>
```

Comment:
There should be absolutely NO logic in views. >Views are only for displaying information that becomes available to it from its corresponding controller not generating it. 
* [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)
* [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright
* Another great article to guide you, [Rails View Helpers Best Practices Guide](https://medium.com/@acushla4real/rails-view-helpers-best-practices-guide-d7d7b6db3db4)